src/ar1D.jl

```diff
@@ -452,14 +452,21 @@ function reinterp_spectra(fname; wavecal_type = "waveCalSkyLine")
         chipWaveSoln = f["chipWaveSoln"]
         close(f)
     else #this is a terrible global fallback, just so we get something to look at
-        chipWaveSoln = zeros(N_XPIX, N_FIBERS, N_CHIPS)
-        for (chipind, chip) in enumerate(["a", "b", "c"])
-            chipWaveSoln[:, :, chipind] .= rough_linear_wave.(
-                1:N_XPIX, a = roughwave_dict[tele][chip][1], b = roughwave_dict[tele][chip][2])
+        if isnothing(backupWaveSoln)
```

Comment:
not that this is any worse than my terrible initial fall back, but have we started thinking about dither shifts in this context yet?